From b5bdd77ff2c8532365806bdfae490b59295fd795 Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Mon, 3 Sep 2012 10:50:22 -0400
Subject: [PATCH 2/2] Use g_thread_try_new rather than g_thread_create

The latter is deprecated. This seems like an extremely worthwhile change
to have to make.
---
 Makefile                 |    2 +-
 c-sources/pcap-monitor.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: b/c-sources/pcap-monitor.c
===================================================================
--- a/c-sources/pcap-monitor.c
+++ b/c-sources/pcap-monitor.c
@@ -444,7 +444,7 @@
       return FALSE;
     }
 
-  priv->thread = g_thread_create (log_thread, &priv->td, TRUE, error);
+  priv->thread = g_thread_try_new ("monitor", log_thread, &priv->td, error);
   if (priv->thread == NULL)
     {
       g_prefix_error (error, "Couldn't spawn logging thread: ");
